### PR TITLE
Style/cards

### PR DESCRIPTION
**Pull request by EthanDuvall**
# Pull Request

## Description
This will update the sytling for the cards  

## Changes Made
- Changed cards to be circles with the button being a heart
- Made the button change color to provide userfeedback when clicked
## Checklist
- [x] The code follows the project's coding standards.
- [ ] Unit tests have been added or updated to cover the changes.
- [ ] Documentation has been updated to reflect the changes (if applicable).
- [x] The code compiles without errors.
- [x] The changes have been tested locally and pass all relevant tests.
- [ ] All new and existing tests pass.
- [x] The pull request has been reviewed by at least one other contributor.

### CODE REVIEW

**Review by AsherSpurr — APPROVED** [review]
Perfection

File: src/components/home/Home.tsx [hunk 1]
```diff
@@ -27,14 +26,14 @@ function Home({ isRedirected, setIsRedirected }: Props) {
...
+            <p className="info-desc">
               Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
-              ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
-              aliquip ex ea commodo consequat.
+              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+              enim ad minim veniam, quis nostrud exercitation ullamco laboris
+              nisi ut aliquip ex ea commodo consequat.
             </p>
             <button className="infoButton">
```
Comment (tednaphil) [inline src/components/home/Home.tsx:35]: Could you remove this button from this Home section?

File: src/components/teas/Teas.tsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ function Teas({ setIsRedirected }: Props) {
 
     function addFavs(newFav: Tea ) {
         console.log(newFav)
-        if(favs.includes(newFav)){
+        if(favs.some(fav => fav.slug === newFav.slug)){
```
Comment (tednaphil) [inline src/components/teas/Teas.tsx:46]: Nice fix!

File: src/components/teas/Teas.tsx [hunk 3]
```diff
@@ -73,9 +73,12 @@ function Teas({ setIsRedirected }: Props) {
     })
 
     return (
+        <>
+        <h2>{category}</h2>
```
Comment (tednaphil) [inline src/components/teas/Teas.tsx:77]: Fabulous to have the category displayed in this component too - good thinking!

**Review by tednaphil — CHANGES_REQUESTED** [review]
See note about the unused button in Home section - thank you for all your work on those fav buttons!

**Review by tednaphil — APPROVED** [review]

File: src/components/home/Home.tsx (hunk 1 above)
Comment (AsherSpurr) [reply src/components/home/Home.tsx:35]: I just removed that in my branch!

**Pull request merged**